src/components/releaseView/ReleaseView.jsx

```diff
@@ -22,6 +24,9 @@ import ReleaseH2 from "../shared/ReleaseH2";
 const ReleaseView = props => {
   const [fullNotes, setFullNotes] = useState([]);
   const [denseNotes, setDenseNotes] = useState([]);
+  const [formattedDate, setFormattedDate] = useState(
+    "Her mangler det en dato.."
```

Comment:
Heller ha en tom string?

Reply:
Ka gir den absolutt mest fantastike brukeropplevelsen? "" eller "Beklager, her mangler det en dato"

Reply:
**Beklager, her mangler det en dato**
Pros: Vise at oss utviklera e ydmyk
Cons: Brukeren bli sur for dei ser der e en feil

**""""**
Pros: Brukeren blir ikkje sur, for dei veit ikkje at noke e feil
Cons: ??

Reply:
Okei, vi kan gå førr plan "Kost det unnjer matta"